sweepbatcher/sweep_batcher.go

```diff
@@ -535,13 +535,15 @@ func (b *Batcher) Run(ctx context.Context) error {
 		case sweepReq := <-b.sweepReqs:
 			sweep, err := b.fetchSweep(runCtx, sweepReq)
 			if err != nil {
-				log.Warnf("fetchSweep failed: %v.", err)
+				log().Warnf("fetchSweep failed: %v.", err)
```

Comment:
Same comment as before, although there it's just one function call, but the wrapped log might be a bit nicer.

Reply:
Added private functions `debuf`, `infof`, `warnf`, and `errorf`.